app/views/contexts/calculated_summary_context.py

```diff
@@ -190,7 +196,7 @@ def _get_formatted_total(
 
     def _get_answer_format(
         self, groups: list, current_location: Location
-    ) -> Tuple[Mapping, list]:
+    ) -> Tuple[Mapping[str, Any], list]:
```

Comment:
Same here, needs to be dict as that is what we actually return.

Reply:
👍